Envfile

```diff
@@ -21,6 +21,8 @@ variable :LOGO_SVG, :String, default: ""
 # Default staff account's id
 variable :DEVTO_USER_ID, :Integer, default: 1
 
+#Default site email
+variable :DEFAULT_SITE_EMAIL, :String, default: "yo@dev.to"
```

Comment:
Could you add a space after the `#` and an empty line after line 25?

Reply:
solved